examples/helloworld/main.go

```diff
@@ -20,59 +20,88 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"math/rand"
 	"time"
 
+	"go.opencensus.io/examples/exporter"
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"go.opencensus.io/trace"
+)
+
+var (
+	// frontendKey allows us breakdown the recorded data
```

Comment:
s/allows us breakdown/allows us to breakdown/g

Reply:
Done